persistence-api/src/main/java/io/stargate/db/tracing/TracingData.java

```diff
@@ -0,0 +1,22 @@
+package io.stargate.db.tracing;
+
+import io.stargate.db.datastore.Row;
+import java.util.List;
+
+public class TracingData {
+  private final List<Row> events;
```

Comment:
We could create a structure similar to java-driver: https://github.com/datastax/java-driver/blob/4.x/core/src/main/java/com/datastax/oss/driver/internal/core/cql/DefaultTraceEvent.java
However, we still would need to map it to the `gRPC` structure, so I think it's better to propagate rows and let the caller do the actual mapping. wdyt?